03-events-forms-refs-life-cycle/life-cycle/giphy-api/src/NewGiphyForm.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import "./NewGiphyForm.css";
+// import jQuery
+import $ from './../node_modules/jquery/dist/jquery.js'; 
```

Comment:
Can you do this with axios or fetch and not jQuery? You should also be able to do import $ from 'jquery'